src/platform/gui.h

```diff
@@ -382,7 +382,7 @@ FileDialogRef CreateSaveFileDialog(WindowRef parentWindow);
 std::vector<Platform::Path> GetFontFiles();
 void OpenInBrowser(const std::string &url);
 
-std::vector<std::string> InitGui(int argc, char **argv);
+std::vector<std::string> InitGui(int& argc, char **argv);
```

Comment:
Why `int& argc`?

Reply:
The abstraction layer gets in the way of normal QApplication usage where the main argc and argv are edited to remove any Qt specific options.  See https://doc.qt.io/qt-5/qguiapplication.html#QGuiApplication

Reply:
When I have some time I'll install QT 5 and try to clean this up. It should not be a problem to do it, but I do not want to give suggestions without a compiler :-)

Reply:
There's so little going on in entrygui.cpp that it seems cleaner to just have each platform provide this.   A pure Qt version would be:

```
int main(int argc, char** argv) { 
    QApplication app(argc, argv);
    Platform::Open3DConnexion();
    SS.Init();
    
    if(argc >= 2) {
        if(argc > 2) {
            dbp("Only the first file passed on command line will be opened.");
        }
        SS.Load(argv[1]);
    }

    app.exec();

    Platform::Close3DConnexion();
    SS.Clear();
    SK.Clear();
    return 0;
}
```

This eliminates InitGui(), RunGui(), ClearGui() and the argument string vector.